src/components/PopupWithConfirmation.js

```diff
@@ -0,0 +1,23 @@
+import Popup from "./Popup";
+
+class PopupWithConfirmation extends Popup {
+  constructor({ popup }, config) {
+    super({ popup });
+    this._popupForm = this._popup;
```

Comment:
`_popupForm` is not a popup

Please,delete it

You need to use ` this._popup` directly in line 16 without defining any ` this._popupForm`